pkg/apiserver/apiserver.go

```diff
@@ -254,8 +254,8 @@ func AddApiWebService(container *restful.Container, apiPrefix string, versions [
 }
 
 // Adds a service to return the supported api versions at /apis.
-func AddApisWebService(container *restful.Container, apiPrefix string, groups []unversioned.APIGroup) {
-	rootAPIHandler := RootAPIHandler(groups)
+func AddApisWebService(container *restful.Container, apiPrefix string, f func() []unversioned.APIGroup) {
```

Comment:
It does change after initialization (this code has already been reviewed and merged elsewhere)